test/e2e/specs/create-voucher-flow.e2e.ts

```diff
@@ -0,0 +1,103 @@
+class CreatePage {
+  get createButton() {
+    return $(
+      "body > div.top_page_container > div.NumPad_numpad__yb7Im > div > button:nth-child(5)"
```

Comment:
those tests will be very flacky. we should use an id instead (I guess, never really made test for browser, but I would assume it's the same as for mobile).